### PR TITLE
[4.0] com_privacy requests

### PR DESCRIPTION
**Pull request by brianteeman**
Updates the tmpl for the requests view to match J4 standards and styles

To test it will be easier if you have some data. These three rows should cover all the options - obviously replace jos_ with your own db prefix

```
INSERT INTO `jos_privacy_requests` (`id`, `email`, `requested_at`, `status`, `request_type`, `confirm_token`, `confirm_token_created_at`) VALUES (1, 'one@example.com', '2019-07-29 11:18:14', 1, 'export', '$2y$10$8mubLEGe/27np9z4w3FRSeJDbvR1rLLd0ljelfxKckYge7fE2COdO', '2019-07-29 11:18:14');
INSERT INTO `jos_privacy_requests` (`id`, `email`, `requested_at`, `status`, `request_type`, `confirm_token`, `confirm_token_created_at`) VALUES (2, 'two@example.com', '2019-07-29 11:18:49', 0, 'export', '$2y$10$q91CGg0h1apxAVFJkyJ/xugIZTZ491YPF859Vl5n1uGCzoeIOOb6S', '2019-07-29 11:18:49');
INSERT INTO `jos_privacy_requests` (`id`, `email`, `requested_at`, `status`, `request_type`, `confirm_token`, `confirm_token_created_at`) VALUES (3, 'three@example.com', '2019-05-29 20:34:19', 1, 'remove', '$2y$10$buYqWYjNr5hrL2P374RcA.hM8WtM8Z0NEEsbe.EgMiB9b9J1l14Y6', '2019-05-29 20:34:19');

```
Other parts of com_privacy are beyond the scope of this PR

@SharkyKZ can you check please if these really are links or should be buttons?

### CODE REVIEW

File: administrator/components/com_privacy/tmpl/requests/default.php [hunk 1]
```diff
@@ -37,88 +35,88 @@
...
-		<div class="clearfix"> </div>
 		<?php if (empty($this->items)) : ?>
 			<div class="alert alert-info">
 				<span class="fa fa-info-circle" aria-hidden="true"></span><span class="sr-only"><?php echo Text::_('INFO'); ?></span>
 				<?php echo Text::_('JGLOBAL_NO_MATCHING_RESULTS'); ?>
 			</div>
 		<?php else : ?>
-			<table class="table table-striped" id="requestList">
+			<table class="table" id="requestList">
+				<caption id="captionTable" class="sr-only">
```
Comment (wilsonge) [inline administrator/components/com_privacy/tmpl/requests/default.php:45]: Redirects?

**Review by wilsonge — CHANGES_REQUESTED** [review]

**Comment by wilsonge** [comment]
Thanks!

**Comment by mbabker** [comment]
> can you check please if these really are links or should be buttons?

The first implementation had them as buttons that submitted the standard form for the list view.  The JavaScript API couldn't cope with actions that didn't result in a navigation (new page loading, reloading the current view, etc.).  So the buttons were changed to "plain" links to bypass the form and the problems that caused.  Ref, https://github.com/joomla-projects/privacy-framework/commit/60b8c2969fa02aa23760dfcc727b1a889058a2f8#diff-2effc18b36deb5dba68915cd87e0178f

If anyone has any bright ideas, they should be applied because right now a lot of the JavaScript API just does not work outside of the very specific workflow of submitting a form and triggering a navigation action.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**